src/applications/vaos/containers/ExpressCareFormPage.jsx

```diff
@@ -1,69 +1,182 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { connect } from 'react-redux';
+import { scrollAndFocus } from '../utils/scrollAndFocus';
+import SchemaForm from 'platform/forms-system/src/js/components/SchemaForm';
 import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
-import { FETCH_STATUS, EXPRESS_CARE_ERROR_REASON } from '../utils/constants';
+import phoneUI from 'platform/forms-system/src/js/definitions/phone';
+import {
+  FETCH_STATUS,
+  EXPRESS_CARE_REASONS,
+  EXPRESS_CARE_ERROR_REASON,
+} from '../utils/constants';
 import FormButtons from '../components/FormButtons';
+import ExpressCareReasonField from '../components/ExpressCareReasonField';
 
 import * as actions from '../actions/expressCare';
 
+const pageKey = 'form';
+const pageTitle = 'Select a reason for your Express Care request';
+
+const initialSchema = {
+  type: 'object',
+  required: ['reasonForRequest', 'phoneNumber', 'email'],
+  properties: {
+    'view:textObject': {
+      type: 'object',
+      properties: {},
+    },
+    reasonForRequest: {
+      type: 'object',
+      required: ['reason'],
+      properties: {
+        reason: {
+          type: 'string',
+        },
+        additionalInformation: {
+          type: 'string',
+        },
+      },
+    },
+    'view:textObject2': {
+      type: 'object',
+      properties: {},
+    },
+    phoneNumber: {
+      type: 'string',
+      pattern: '^[0-9]{10}$',
+    },
+    email: {
+      type: 'string',
+      format: 'email',
+    },
+  },
+};
+
+const uiSchema = {
+  'view:textObject': {
+    'ui:description': (
+      <>
+        <h2>Tell us about your health concern</h2>
+        <p>
+          Please select a concern from the list the best fits your needs today.
+          <span className="schemaform-required-span">(*Required)</span>
+        </p>
+      </>
+    ),
+  },
+  reasonForRequest: {
+    'ui:field': ExpressCareReasonField,
+    'ui:description': <h3>text</h3>,
+    options: {
+      items: EXPRESS_CARE_REASONS.map((r, index) => ({
+        id: `express-care-reason-${index}`,
+        value: r.reason,
+        label: r.reason,
+        secondaryLabel: r.secondaryLabel,
+      })),
+    },
+  },
+  'view:textObject2': {
+    'ui:description': (
+      <>
+        <h2>Share your contact information</h2>
```

Comment:
I believe this header should be a `legend` inside a `fieldset` for the phone and email fields, since this is a title for that group of fields. 

The typical way to do that within the form system is to make make some kind of `contactInformation` object, which has a `ui:title` and `ui:description` with these two pieces of text, and the `phoneNumber` and `email` fields within that same object. 

When the form system sees an `object` schema type with a title, it wraps the fields in that object in a `fieldset` and puts the title in a `legend`.

Reply:
Updated